README.md

```diff
@@ -1 +1,50 @@
-# Hello-world
\ No newline at end of file
+# Hello world
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## Built With
+
+- Html
+- Css
+- Linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
Kindly consider these instructions right here as they tell you how to use this template.